Controller/CRUDController.php

```diff
@@ -362,7 +362,6 @@ public function editAction($id = null)
         $form->handleRequest($request);
 
         if ($this->getRestMethod() == 'POST') {
-            $form->submit($request);
```

Comment:
Nope, `handleRequest` is the good way and it's available since SF 2.3.

But the `if ($this->getRestMethod() == 'POST')` condition should be removed too.